Code/Editor/ViewportManipulatorController.h

```diff
@@ -46,5 +48,8 @@ namespace SandboxEditor
         AZStd::unordered_map<AzToolsFramework::ViewportInteraction::MouseButton, ClickEvent> m_pendingDoubleClicks;
 
         AZ::ScriptTimePoint m_currentTime;
+        
+        bool m_captureStart;
+        AZ::Vector2 m_virtualPosition;
```

Comment:
update: Maybe this should be called `m_virtualNormalizedPosition` to make it clear it's in the 0-1 range?